lib/services/cordova-migration-service.ts

```diff
@@ -107,8 +117,25 @@ export class CordovaMigrationService implements ICordovaMigrationService {
 			});
 
 			let supportedPlugins = this.pluginsForVersion(toVersion).wait();
-			plugins = _.filter(plugins, plugin => _.contains(supportedPlugins, plugin));
+			plugins = _.filter(plugins, plugin => {
+				if (plugin.indexOf('@') > -1) {
+					let pluginBasicInfo = this.$pluginsService.getPluginBasicInformation(plugin);
+					if (this.isPluginSupported(pluginBasicInfo.name, pluginBasicInfo.version, toVersion)) {
+						supportedPlugins.push(plugin);
```

Comment:
shouldn't you just return true here instead of adding the plugin to supportedPlugins and at the end check it the plugin exists in supportedPlugins?